src/ci/docker/x86_64-gnu-llvm-7/Dockerfile

```diff
@@ -28,7 +28,7 @@ ENV RUST_CONFIGURE_ARGS \
       --enable-llvm-link-shared \
       --set rust.thin-lto-import-instr-limit=10
 
-ENV SCRIPT python2.7 ../x.py test src/tools/tidy && python2.7 ../x.py test
+ENV SCRIPT python2.7 ../x.py test
```

Comment:
This won't work. Tidy is already the first thing in the bootstrap list. We'll instead want to do something like this. (We can let the PR builder confirm that I'm doing the right thing).

```
../x.py test --exclude src/tools/tidy && ../x.py test src/tools/tidy
```

Reply:
Ah, I didn't know that, addressed.